app/scripts/lib/multichain-api/caip25permissions.test.ts

```diff
@@ -99,4 +101,53 @@ describe('endowment:caip25', () => {
       });
     });
   });
+  describe('caveat mutator removeAccount', () => {
+    it('can remove an account', () => {
```

Comment:
[nit] should we do a test for an account that is permissioned across multiple scopes